src/Symfony/Component/Lock/Store/DatabaseTableTrait.php

```diff
@@ -50,7 +50,7 @@ private function init(array $options, float $gcProbability, int $initialTtl): vo
      */
     private function getHashedKey(Key $key): string
     {
-        return hash('sha256', (string) $key);
+        return hash('xxh128', (string) $key);
```

Comment:
I'd keep this one as is because the key might contain input data

Reply:
OK. I reverted this and the other occurrence. Thanks.